util/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+//! The `util` package contains useful structs, traits, types, etc. that can be easily used across
+//! all the Witnet-rust project.
+
```

Comment:
add `#![deny(rust_2018_idioms)]` to force rust_2018, and let `cargo fix` help you to translate the patch.